llama_index/llms/azure_openai.py

```diff
@@ -1,10 +1,15 @@
 from typing import Any, Dict, Optional
 
+from pydantic import root_validator
+
 from llama_index.bridge.pydantic import Field
 
 from llama_index.callbacks import CallbackManager
 from llama_index.llms.openai import OpenAI
 
+AZURE_OPENAI_API_TYPE = "azure"
+AZURE_OPENAI_VERSION = "2023-05-15"
+
```

Comment:
Super minor -- Should the default azure version be something that supports function calling? It was added in the July and August preview I think.

Reply:
Good call, I keep getting confused on which one is the recommend/latest.